python/tvm/contrib/target/android_nnapi/relayir_to_nnapi_converter/_export_object/helper.py

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Namespace for helper objects/methods that's not part of the JSON
+content. This includes the symbol table, checking methods, ...
+"""
+from .operand import Operand as _Operand
+
+
+class Helper:
```

Comment:
`Helper` is not a good name and should be improved.

Reply:
Perhaps I can name it "analysis"?
This namespace (for now) includes read-and-compute methods that help checking for some properties during codegen.

Reply:
analysis.py:XXAnalyzer seems much better to me.

Reply:
I would make it "JSONAnalyser" then. :)